ports/nrf/mphalport.c

```diff
@@ -129,7 +129,10 @@ static void rtc_irq_time(nrfx_rtc_int_type_t event) {
 
 void rtc1_init_time_ticks(void) {
     // Start the low-frequency clock (if it hasn't been started already)
-    mp_nrf_start_lfclk();
+    if (!nrf_clock_lf_is_running(NRF_CLOCK)) {
+        nrf_clock_task_trigger(NRF_CLOCK, NRF_CLOCK_TASK_LFCLKSTART);
+        rtc_offset_check();
+    }
```

Comment:
Please don’t undo my change from b10182b, it was done for a reason.

Reply:
That was probably a rebasing error, I'll put it back. Thanks!